src/hotkey/hotkey_item.cpp

```diff
@@ -157,6 +157,7 @@ bool hotkey_base::matches(const SDL_Event &event) const
 void hotkey_base::save(config& item) const
 {
 	item["command"] = get_command();
+	item["disabled"] = is_disabled() ? config::attribute_value::s_true : config::attribute_value::s_false;
```

Comment:
So I am still yet to know all the nuances of how Wesnoth works got the names mixed up. Yes, I was referring to the configuration attributes. I already understand the principle behind using booleans directly, I simply did not know that this is how the configuration attributes were designed.

Again, it's more helpful to explain how things can be done better and correct misunderstandings, rather than what seems like being told 'you're wrong'. Thanks.